### DISCUSSION TITLE
My opinionated fork

### DISCUSSION THREAD

**Discussion by ivan-avalos** [daneren2005/Subsonic#1135] [Show and tell]
Hey, @daneren2005! I forked this amazing app to create my own opinionated version. I'm making some UI customization so it looks prettier (to me). I revamped the Now Playing UI, and replaced all the icons with rounded XML ones. Take what you want from it!

**Source code:** https://git.avalos.me/dsub

_**Note:** I don't promise long-term maintenance of this fork._

![Screenshot_20221213-213038_DSub](https://user-images.githubusercontent.com/16617314/207500524-180e3298-2aae-4a3f-8fb2-e97933de5957.png)

**Comment by gorman42** [comment]
Any chance for a usable build?
